.github/issue_template.md

```diff
@@ -1,15 +1,2 @@
-### Description
+# Description
 
-[Description of the issue or feature]
-
-### Research/based on
-
-[Which codebases, pallets or designs did you base your design on]
-
-### How will this affect the code base
-
-[Does it improve readability, performance? Will it introduce new complexity?]
-
-### What are foreseen obstacles or hurdles to overcome?
-
-[Are migrations needed, structural changes around testing etc?]
```

Comment:
I like that sections TBH and found it helpful to have the structure.

Reply:
my POV is that these are often good questions to ask oneself when creating an issue but I personally don't find value is having to try and fit this structure per se. my 2cents, whatever is more helpful to the team is good with me 👍

Reply:
IMO a section for description and probably another one for extra notes might be more than enough.

Reply:
I am good with what the matority wants

Reply:
I do not have a strong opinion of these sections. It's true that sometimes they do not apply to the issue or can be handled by tags (as "migration needed"), but they also help to explain the issue better

Reply:
Personally, I would like to keep this but it seems like the majority choses to ignore it. I don't have a strong opinion.

Reply:
@wischli maybe we can find a middle ground that makes everyone happy?